### ISSUE TITLE
 'opt_background' latent input should be used with the IC-Light 'fbc' model

### ISSUE THREAD

**Issue by gtbloody** [kijai/ComfyUI-IC-Light#58]
![image](https://github.com/user-attachments/assets/e2d89342-646e-4d72-8ae4-45f11725f87a)
【Input channels 8 does not match model in_channels 12, 'opt_background' latent input should be used with the IC-Light 'fbc' model, and only with it】
I guess comfyui's official update broke this again
the last time was https://github.com/kijai/ComfyUI-IC-Light/issues/51

**Comment by kijai** [comment]
I just updated Comfy and it is still working fine, so nothing has broken there. The error you're getting is because when using the fbc model you need to input the background latent in the opt_background -input, and when using the fc -model, it shouldn't be connected at all.

**Comment by vincentw3** [comment]
Me too,I have the same problem

**Comment by gtbloody** [comment]
> Me too,I have the same problem

Restructure your workflow

**Comment by Xshinto** [comment]
I'm having the same issue, I downloaded all the model "fc, fbc" models. Updated the comfyUI. and still the same issue.

**Comment by Xshinto** [comment]
> I just updated Comfy and it is still working fine, so nothing has broken there. The error you're getting is because when using the fbc model you need to input the background latent in the opt_background -input, and when using the fc -model, it shouldn't be connected at all.

Yes it's not connected in my case. and I have the 'fc' model chosen since im only focused on the foreground. but still getting the same issue..

**Comment by Xshinto** [comment]
FIXED - What fixed for me was to re-download the VAE. I was using "AE" initially. Then I downloaded vae-ft-mse-840000-ema-pruned" and fixed the problem! 

https://huggingface.co/EvilEngine/vae-ft-mse-840000-ema-pruned/tree/main
